app/mailers/conversation_reply_mailer.rb

```diff
@@ -40,7 +40,9 @@ def reply_without_summary(conversation, last_queued_id)
            reply_to: reply_email,
            subject: mail_subject,
            message_id: custom_message_id,
-           in_reply_to: in_reply_to_email
+           in_reply_to: in_reply_to_email,
+           cc: cc_bcc_emails[0],
+           bcc: cc_bcc_emails[1]
```

Comment:
@santhosh-chinnasamy Was this change intentional? It looks like the change is not related to the PR.

Reply:
it was linted by rubocop